Library/Formula/mplayer.rb

```diff
@@ -8,9 +8,11 @@ class Mplayer < Formula
   end
 
   bottle do
-    sha256 "5504434dce05f51d0d6336426f64af559740203193f6f34d1ef250ebfe955d8f" => :el_capitan
-    sha256 "6d6ccf0b963bc4a0edd48bc7d1e1008db888dc38b91fe85c6c58c5d6dded4f48" => :yosemite
-    sha256 "d90208d62889b8c74d06917ba926f91fddb039ab15b27c825e4356b75b9d105b" => :mavericks
+    revision 1
```

Comment:
eh sure, just not sure why as I thought the bot will update it anyway... Will do the commit later.
What is the reasoning behind restoring this part? Just for cleanlyness' sake? ;-)